code/modules/crafting/anvil_recipes/armor.dm

```diff
@@ -165,11 +177,21 @@
 	created_item = /obj/item/clothing/neck/bevor
 	craftdiff = 2
 
+/datum/anvil_recipe/armor/steel/nasal_helmet
+	name = "Nasal helmet"
+	recipe_name = "a Nasal helmet"
+	req_bar = /obj/item/ingot/steel
+	created_item = /obj/item/clothing/head/helmet/nasal
+	createmultiple = TRUE
```

Comment:
If you're making them craftable by batches (this is making TWO nasal helmets, read the code comments on these vars please), then `null` nasal helmet's smeltresult as well, or change the name of this recipe to add (x2) and its recipe name to "a couple Nasal Helmets".
If you want this to craft only one helmet, then this var should be FALSE, and the createditem_num var erased.